cloud: support generating endpoints automatically (#14244)
pingcap/docs: tidb-cloud/tidb-cloud-release-notes.md

This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2023.

> ## July 20, 2023

Author: ```suggestion
## July xx, 2023
```

This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2023.

## July 20, 2023

**General changes**

- Introduce a powerful feature in TiDB Cloud [Data Service](https://tidbcloud.com/console/data-service) - Automatic Endpoint Generation.

    Developers can now effortlessly create HTTP endpoints with minimal clicks and configurations. Eliminate repetitive boilerplate code, simplify and accelerate endpoint creation, and reduce potential errors.

>     This feature empowers developers, enabling them to build backend solutions without the need to learn any programming language. Seamlessly unlock new possibilities for application development.
> 

Author: ```suggestion
```

This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2023.

> ## July 27, 2023

Author: ```suggestion
## July 26, 2023
```

**General changes**

> - Introduce a powerful feature in TiDB Cloud [Data Service](https://tidbcloud.com/console/data-service) - Automatic Endpoint Generation.

Reviewer: ```suggestion
- Introduce a powerful feature in TiDB Cloud [Data Service](https://tidbcloud.com/console/data-service): Automatic endpoint generation.
```

    Developers can now effortlessly create HTTP endpoints with minimal clicks and configurations. Eliminate repetitive boilerplate code, simplify and accelerate endpoint creation, and reduce potential errors.

    For more information on how to use this feature, see [Generate an endpoint automatically](/tidb-cloud/data-service-manage-endpoint.md#generate-an-endpoint-automatically).

- Support `PUT` and `DELETE` request methods for endpoints in TiDB Cloud [Data Service](https://tidbcloud.com/console/data-service).

>     - You can use the `PUT` method to update or modify data, such as an `UPDATE` statement.

Reviewer 2: Use the PUT method to update or modify data, similar to an UPDATE statement.
Use the DELETE method to delete data, similar to a DELETE statement.

    - Use the `DELETE` method to delete data, similar to a `DELETE` statement.

>     For more information, see [Configure properties](/tidb-cloud/data-service-manage-endpoint.md#configure-properties).

Reviewer 3: ```suggestion
  For more information, see [Configure properties](/tidb-cloud/data-service-manage-endpoint.md#configure-properties).
```

- Support **Batch Operation** for `POST`, `PUT`, and `DELETE` request methods in TiDB Cloud [Data Service](https://tidbcloud.com/console/data-service).

>     When **Batch Operation** is enabled for an endpoint, you can operate on multiple rows in a single request. For example, you can insert multiple rows of data in a single `POST` request.

Reviewer 2: When Batch Operation is enabled for an endpoint, you gain the ability to perform operations on multiple rows in a single request. For instance, you can insert multiple rows of data using a single POST request.

>     When **Batch Operation** is enabled for an endpoint, you can operate on multiple rows in a single request. For example, you can insert multiple rows of data in a single `POST` request.

Author: ```suggestion
    When Batch Operation is enabled for an endpoint, you gain the ability to perform operations on multiple rows in a single request. For instance, you can insert multiple rows of data using a single POST request.
```
